FirebaseStorage/Tests/Unit/FIRStorageTestHelpers.m

```diff
@@ -13,8 +13,9 @@
 // limitations under the License.
 
 #import "FirebaseStorage/Tests/Unit/FIRStorageTestHelpers.h"
+#import "FirebaseStorage/Sources/Public/FirebaseStorage/FIRStorage.h"
 
-#import "FirebaseStorage/Sources/FIRStorageComponent.h"
+//#import "FirebaseStorage/Sources/FIRStorageComponent.h"
```

Comment:
Should this commented out import be removed? 
```suggestion
```

Reply:
Yes.